CHANGELOG.rst

```diff
@@ -27,6 +27,7 @@ Added
   (``rasa.core.agent.handle_channels()``). The number of workers can be set using the
   environment variable ``SANIC_WORKERS`` (default: 1). A value of >1 is allowed only in
   combination with ``RedisLockStore`` as the lock store.
+- Updated the /status api route to use the actual model file location instead of the tmp location.
```

Comment:
I think this go in ``changed`` and not in `added`
```suggestion
- Updated the ``/status`` api route to use the actual model file location instead of the ``tmp`` location.
```

Reply:
sorry didn't notice I put it in the wrong area, just resolved it.

Reply:
Ok let me check I thought it already had tests for this since I'm just updating the same model, isn't this the test here?

https://github.com/RasaHQ/rasa/blob/fix_model_status/tests/test_server.py#L110